inc/parser/xhtml.php

```diff
@@ -1108,7 +1108,13 @@ function emaillink($address, $name = null, $returnonly = false) {
     function internalmedia($src, $title = null, $align = null, $width = null,
                            $height = null, $cache = null, $linking = null, $return = false) {
         global $ID;
-        list($src, $hash) = explode('#', $src, 2);
+
+        if (strpos($src, '#') !== false) {
+            list($src, $hash) = explode('#', $src, 2);
+        } else {
+            $hash = null;
+        }
+
```

Comment:
If I would see a limit I usually would not expect if it set it to 2 to always get two results.
I'd expect to get 2 or less results since it defined a max limit.

Suppressing would not show the notice but will take more time to parse. Personally I would not suggest this, not even as a temp solution.